docs/api/api/api.txt

```diff
@@ -151,6 +151,8 @@ XmlResult: status
 
 Parameters:
   cmd: change_password
+  cmd: lock             # will lock the user and his home projects
+  cmd: delete           # will mark the user as deleted and remove her home projects
```

Comment:
Why do we need to mark them as deleted instead of actually destroy the user? I mean if you actually removes the home project... why it make sense to keep the user?